dev/breeze/src/airflow_breeze/utils/packages.py

```diff
@@ -420,7 +420,12 @@ def get_dist_package_name_prefix(provider_id: str) -> str:
 
 
 def apply_version_suffix(install_clause: str, version_suffix: str) -> str:
-    if install_clause.startswith("apache-airflow") and ">=" in install_clause and version_suffix:
+    if (
+        install_clause.startswith("apache-airflow")
+        and ">=" in install_clause
+        and version_suffix
+        and not is_local_version_suffix(version_suffix)
```

Comment:
Hmm. I think we should support BOTH pypi and local suffix. It's really likely that we want to have 

rc1+local-tag

Reply:
I changed this to ignore the local suffix, if any, and process this if there is a PyPi suffix.

Reply:
Looks goood. Indeed - just doc leaving old example would be great.